joplin/api/schema.py

```diff
@@ -274,32 +274,52 @@ class Language(graphene.Enum):
 
 
 class ServicePageNode(DjangoObjectType):
+    page_type = graphene.String()
+
     class Meta:
         model = ServicePage
         filter_fields = ['id', 'slug', 'live', 'coa_global']
         interfaces = [graphene.Node]
 
+    def resolve_page_type(self, info):
+        return "service page"
+
 
```

Comment:
The page_type for Service, Forms, Guides, Information, Department and Official Documents are all hardcoded this way with their page type name like this. Obviously this could be refactored a bit. And, will be referenced in an Issue linked in this PR. 
- This change was necessary so that the API tells Janis what "icon" to show for what page it is!

Reply:
Can we get the name from the model directly instead of hardcoding the page types? Ex: `ServicePage.get_verbose_name().lower()`